packages/node/example/multiSession/src/serverSideApp.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2021 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+// The only import we need from the Node AuthN library is the Session class.
+import {
+  getSessionFromStorage,
+  Session,
+  getSessionIdFromStorageAll,
+} from "../../../dist/Session";
+
+import cookieSession from "cookie-session";
+
+const clientApplicationName = "solid-client-authn-node multi session demo";
+
+import express from "express";
+const app = express();
+const PORT = 3001;
+
+const DEFAULT_OIDC_ISSUER = "https://broker.pod.inrupt.com/";
+
+app.use(
+  cookieSession({
+    name: "session",
+    // These keys are required by cookie-session to sign the cookies.
+    keys: ["key1", "key2"],
```

Comment:
So are the values of these keys relevant now? If not, I'd use their values to explain that they are required, but not used in this demo, and so their values are irrelevant
```suggestion
    // These keys are required by cookie-session to sign the cookies, but we don't use that signing feature in this demo.
    keys: ["Required, but value not relevant - key1", "Required, but value not relevant - key2"],
```

Reply:
Express does sign the cookies though, but it's all handled by `cookie-session`

Reply:
Ok fine, so then I'd just change the values to reflect that:
```suggestion
    // These keys are required by cookie-session to sign the cookies.
    keys: ["Required, but value not relevant for this demo - key1", "Required, but value not relevant for this demo - key2"],
```